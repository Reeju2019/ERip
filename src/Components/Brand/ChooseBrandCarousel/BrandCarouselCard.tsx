import * as React from 'react'
import './ChooseBrandCarousel.css'

interface IBrandCarouselCardProps {
  data: {
    brandName: string
    image: string
  }
}

const BrandCarouselCard: React.FunctionComponent<IBrandCarouselCardProps> = (data) => {
  return (
    <>
      <div>
        <div className='image-containers'>
          <img src={data.data.image} className='image-cbs' alt={data.data.brandName} />
        </div>
      </div>
    </>
  )
}

export default BrandCarouselCard
