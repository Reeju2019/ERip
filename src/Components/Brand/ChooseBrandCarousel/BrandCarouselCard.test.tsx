import { render } from '@testing-library/react'
import BrandCarouselCard from './BrandCarouselCard'
import renderer from 'react-test-renderer'

test('choose brand card', () => {
  render(<BrandCarouselCard />)
})
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => jest.fn(),
}))
describe('App rendering specification', () => {
    it('SnapShot testing', () => {
      const component = renderer.create(<BrandCarouselCard />)
      const tree = component.toJSON()
      expect(tree).toMatchSnapshot()
      
    });
})
