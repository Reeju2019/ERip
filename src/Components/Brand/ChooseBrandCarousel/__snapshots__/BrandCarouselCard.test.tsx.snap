// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div>
  <div
    className="image-containers"
  >
    <img
      className="image-cbs"
    />
  </div>
</div>
`;
